.github/workflows/build.yml

```diff
@@ -41,3 +43,37 @@ jobs:
         with:
           name: ark-gallery.AppImage
           path: ./src-tauri/target/release/bundle/appimage/ark-gallery_*_amd64.AppImage
+
+  build-windows:
+    runs-on: windows-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          components: rustfmt, clippy
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20.x'
+
+      - name: Install dependencies
+        run: |
+          choco install -y nodejs
+
+      - name: Fetch Node.js Dependencies
+        run: yarn
+
+      - name: Build Release version
+        run: yarn tauri build
+
+      - name: Upload App
+        uses: actions/upload-artifact@v2
+        with:
+          name: ark-gallery-Windows.zip
+          path: ./src-tauri/target/release/bundle/
```

Comment:
Should it really handle folder paths? With `AppImage` we provide wildcarded path to the binary itself.